lib/senders/sms.js

```diff
@@ -69,6 +81,50 @@ module.exports = (log, translator, templates, config) => {
     }
   }
 
+  function pollCurrentSpend () {
+    let limit
+
+    sns.getSMSAttributes({ attributes: [ 'MonthlySpendLimit' ] }).promise()
+      .then(result => {
+        limit = parseFloat(result.MonthlySpendLimit)
+        if (isNaN(limit)) {
+          throw new Error(`Invalid MonthlySpendLimit "${result.MonthlySpendLimit}"`)
+        }
+
+        const now = new Date()
+        return cloudwatch.getMetricsStatistics({
+          Namespace: 'AWS/SNS',
+          MetricName: 'SMSMonthToDateSpentUSD',
+          StartTime: startOfMonth(now).toISOString(),
+          EndTime: now.toISOString(),
+          Period: SECONDS_PER_WEEK,
```

Comment:
A big arbitrary period to reduce the number of datapoints we get back from AWS, because we're only interested in the grand total for the month.

Reply:
Seems like ^ is a good comment to put on top of `Period`.

Reply:
I'm sorry, I misled you on the data returned by cloudwatch! 

The Metric `SMSMonthToDateSpentUSD` is a point-in-time "you've spent this much on SMS". So if you just need the latest endpoint you could do `StartTime: now.toISOString() - 5 minutes, EndTime: now.toISOString(), Period: 300, Statistics: 'Average'` to grab the latest endpoint.

You can delete the array reducing code below too, since it's only a single datapoint